src/world.rs

```diff
@@ -211,6 +282,19 @@ impl World {
         self.archetypes[meta.ty as usize].find::<C>().is_some()
     }
 
+    /// Get an immutable reference to the component of the given type for an [Entity](struct.Entity.html).
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use rs_ecs::*;
+    /// let mut world = World::new();
+    ///
+    /// let entity = world.alloc();
+    /// world.insert(entity, (42_u32, true));
+    ///
+    /// let comp = world.get::<u32>(entity);
```

Comment:
`unwrap`ing the component would make this a more powerful test.

Reply:
Same for `get_mut`